data-manager/pom.xml

```diff
@@ -69,6 +69,14 @@
             <groupId>org.orbisgis</groupId>
             <artifactId>postgis-jts-osgi</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
```

Comment:
Why this new logger ?

Reply:
It is used during the test to gather and display the log messages

Reply:
Just a question.
Why not use the same logging framework as in other OrbisGIS projects, log4j ?
Is logback better ?

Reply:
Logback is easier to be configured and as it is aimed to be used only for test purpose, it doesn't have any impact on the other OrbisGIS projects.